core-plugin/testSrc/com/google/gct/idea/appengine/validation/EndpointTestBase.java

```diff
@@ -75,4 +86,24 @@ public void run() {
       }
     }
   }
+
+  private void removeEndpointsSdkFromProject() {
+    ApplicationManager.getApplication().runWriteAction(new Runnable() {
+      @Override
+      public void run() {
+        final LibraryTable table =
```

Comment:
why final? Google style is not to make local varibles final unless required. Is IntelliJ style different?